contracts/estate/EstateRegistry.sol

```diff
@@ -100,7 +90,6 @@ contract EstateRegistry is ERC721Token, Ownable, MetadataHolderBase, IEstateRegi
     returns (bytes4)
   {
     uint256 estateId = _bytesToUint(estateTokenIdBytes);
-    require(exists(estateId), "The estate id should exist");
```

Comment:
What's the reason for removing this check?

Reply:
The reasoning behind it was that the check is already present on `_pushLandId(estateId, tokenId);` which is the method that's using `estateId`. By that I mean that if `_pushLandId` wasn't there, not having a valid `estateId` shouldn't _really_ matter.

Do you think it's better to keep it here too?

Reply:
@NicoSantangelo That makes complete sense, I think that's the right decision to remove the check.

Reply:
@pcowgill Github notifications are a mess and I completely missed this. I think it's a good idea!

We were discussing a similar thing regarding the way we write arguments (with or without `_`) because it's not always consistent.

I think that rewriting the contracts it's not really viable, but I'll discuss it with the team for any feature contracts we do.

Thanks!